dropwizard-jersey/src/test/java/io/dropwizard/jersey/jackson/JsonProcessingExceptionMapperTest.java

```diff
@@ -110,11 +110,6 @@ public void returnsA400ForMalformedListInputCausingIoException() throws Exceptio
         assertEndpointReturns400("urlList", "[\"no-scheme.com\"]");
     }
 
-    @Test
-    public void returnsA400ForNonDeserializableRequestEntities() throws Exception {
-        assertEndpointReturns400("ok", new UnknownRepresentation(100));
-    }
```

Comment:
Any particular reason this test was removed? Should it now return a 200 or something else?

Reply:
This test case seems to in practice test that Dropwizard fails on unknown JSON properties, which is no longer the expected behavior. There also seems to be other test cases covering other variants of non-deserializable request entities.

However, I see now that there's no test testing to not provide a `@NotNull` field. Maybe this test can be rewritten so that it tests that case instead?